_includes/datagov-search-display.html

```diff
@@ -69,10 +69,15 @@
   {{- content -}}
 </div> 
 <div class="search pagination padding--bottom--xl">
-  <span class="sgds-icon sgds-icon-arrow-left is-size-4"></span>
+  <span class="bx bx-chevrons-left is-size-4"></span>
+  <div class="px-1 is-hidden-mobile"></div>
+  <span class="sgds-icon sgds-icon-arrow-left is-size-4 is-hidden-desktop is-hidden-tablet-only"></span>
```

Comment:
Same issue for this arrow as well (although not as important since mobile).

Reply:
https://github.com/isomerpages/isomerpages-template/pull/320/commits/6a4df4a5910c146b4b01463a583e6341af5373ba